ext/src/content/content.js

```diff
@@ -3,6 +3,7 @@
 function init() {
   const GITHUB_URL = "https://github.com";
   const GITHUB_VALID_PATHNAMES = /^\/.*\/.*\/(?:pull\/\d+(?:\/?|\/files\/?)$|commit|compare\/.*|issues\/\d+|issues\/new)/u;
+  const POLLING_INTERVAL = 30;
```

Comment:
Added this back because it's referenced later in the code. Not sure why CI didn't catch this.